src/views/Dashboard.vue

```diff
@@ -4,8 +4,11 @@
 		:show-more-text="title"
 		:loading="isLoading">
 		<template #empty-content>
-			<EmptyContent v-if="emptyContentMessage"
-				:icon="emptyContentIcon">
+			<EmptyContent v-if="emptyContentMessage">
+				<template #icon>
+					<CheckBoldIcon v-if="state === 'ok'" :size="70" />
```

Comment:
```suggestion
					<CheckBoldIcon v-if="state === STATE.OK" :size="70" />
```
please use the state variables from the STATE obj

Reply:
Done in latest push!